Notice/Observable.swift

```diff
@@ -9,12 +9,13 @@ public struct Observable<T> {
     public var value: T {
         didSet {
             dispatch_sync(eventQueue) {
-                self.new.invoke(self.value)
-                self.newOld.invoke((old: oldValue, new: self.value))
+                self.invoke(self.value)
             }
         }
     }
 
+    public var subscriptions = Set<Subscription<T -> Void>>()
```

Comment:
They feel pretty good. Contemplated doing that before, but bailed. You mentioning that now leads me to believe it should be done.